code-challenges.test.js

```diff
@@ -131,7 +131,7 @@ describe("noDoubles", () => {
 //  FAIL  ./code-challenges.test.js
 //  ● noDoubles › takes in two arrays as arguments
       //  ReferenceError: noDoubles is not defined
-
+ 
```

Comment:
JavaScript 1:
Nice job using `.sort()` with `Math.random()` to shuffle the array! A couple notes: you don't need the extra set of curly braces that are holding the sort and return! Also, as a general rule of thumb, if you are creating a variable just to return it on the next line, there's no need to create the variable at all, you can just return the data itself!

JavaScript 2:
Great job using dot notation to access the values at the corresponding keys. Again, no need to make that `newValue` variable, you can just return the logic it's holding!

JavaScript 3:
Nice use of `.filter()` to get rid of dupes! And here you did the right thing and returned the data instead of creating an unnecessary variable, good work!